spec/amber/router/pipe/pipeline_spec.cr

```diff
@@ -67,6 +67,13 @@ module Amber
           response.body.should eq "Destroy"
         end
 
+        it "perform HEAD request" do
```

Comment:
I think this should be `it "responds to HEAD requests"` or `it "provides response for HEAD requests"`

"Performing the request" is more what the client is doing.